pkg/tools/interfaces.go

```diff
@@ -25,6 +25,8 @@ const (
 	EtcdErrorCodeTestFailed    = 101
 	EtcdErrorCodeNodeExist     = 105
 	EtcdErrorCodeValueRequired = 200
+	EtcdErrorCodeWatchExpired  = 401
+	EtcdErrorCodeUnreachable   = 501
```

Comment:
These are the errors returned by Etcd

On Thu, Nov 5, 2015 at 8:49 AM, David Eads <notifications@github.com> wrote:

> In pkg/tools/interfaces.go
> <https://github.com/kubernetes/kubernetes/pull/16807#discussion_r44012447>
> :
>
> > @@ -25,6 +25,8 @@ const (
> >  	EtcdErrorCodeTestFailed    = 101
> >  	EtcdErrorCodeNodeExist     = 105
> >  	EtcdErrorCodeValueRequired = 200
> > +	EtcdErrorCodeWatchExpired  = 401
> > +	EtcdErrorCodeUnreachable   = 501
>
> NotImplemented (501) over ServiceUnavailable (503)?
>
> Oh, these are just random constant values? Can you shift the new ones out
> of the HTTP error code ranges?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/16807/files#r44012447>.
>